### DISCUSSION TITLE
How do we explain the pronounced increase in detections after Nov 1, 2023?

### DISCUSSION THREAD

**Discussion by adrmac** [orcasound/orcasite#538] [General]
I created the data visualization below from the 'detections' endpoint at https://live.orcasound.net/graphiql. This shows the number of detections submitted by day over an 18-month period from Feb 2023 - July 2024, broken out by node. There is a pronounced increase in detections after Nov 1, 2023. This is especially true for the Sunset Bay node, but a similar pattern appears across pretty much all nodes. 
![detection count](https://github.com/user-attachments/assets/9586f023-3f6b-49a0-9476-e7f7ef00c8f2)
https://docs.google.com/spreadsheets/d/1sMpn_WUDNlRq4NE_DTwhHTG98Wy3vyLAIjyTu1lZYMA/edit?gid=847555337#gid=847555337

What do we think could explain this pattern?

What prompted me to look at this data was that Nov 1, 2023 happens to be the day Orcasound launched "V3" interface update designed by the UX team, which added the interactive map view. Could this really have had such a strong effect? What else happened around that time?

**Comment by UXBrendan** [comment]
Good questions, @adrmac. I wish I could provide some insight. As a UX Researcher who was intimately involved in the redesign before it was sent to development, I need to  be able to show impact by increased user engagement. Is there a way to accurately measure this?

**Reply by adrmac** [reply]
@UXBrendan Have you thought about doing a post-launch usability study?

**Reply by UXBrendan** [reply]
@adrmac - That's a good suggestion. I would have to have a clear objective and question that needed to be answered. From my perspective, a quantitative usability study would be the methodology I would recommend running, and those require expensive platforms that cost thousands of dollars to get the N of participants needed to reach statistical significance. 

The question that is easier (and cheaper) to answer is- what is the accurate user engagement with tagged sounds before and after launch? 

This should be answerable by reviewing analytics.

**Comment by skanderm** [comment]
@adrmac as you mentioned, that's when we switched over the design. Another factor may have been an increased number of email notifications and posts to various orca groups during live events. There's probably a much higher amount of engagement during those times.

**Reply by skanderm** [reply]
You might also find interesting information if you compare it with the listener count during those times. Contained in each detection is the `listenerCount` for how many people were listening live at the time. Perhaps if you aggregate the number of detections across a particular span of time during a live event, the ratio might be linear.
